pkgs/development/ocaml-modules/jwto/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, buildDunePackage, fetchFromGitHub, alcotest, cryptokit, fmt, yojson
+, base64, re, ppx_deriving }:
+
+buildDunePackage rec {
+  pname = "jwto";
+  version = "0.3.0";
+
+  minimumOCamlVersion = "4.05";
+
+  src = fetchFromGitHub {
+    owner = "sporto";
+    repo = "jwto";
+    rev = "${version}";
+    sha256 = "1p799zk8j9c0002xzi2x7ndj1bzqf14744ampcqndrjnsi7mq71s";
+  };
+
+  propagatedBuildInputs =
+    [ alcotest cryptokit fmt yojson base64 re ppx_deriving ];
+
+  doCheck = true;
+
+  meta = {
+    homepage = "https://github.com/sporto/jwto";
+    description = "JSON Web Tokens (JWT) for OCaml.";
```

`meta.description` should not end with a period, as per the [contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).
```suggestion
    description = "JSON Web Tokens (JWT) for OCaml";
```